### PR TITLE
Add initial alert guide

### PR DESCRIPTION
**Pull request by darakian**
I wrote this guide after working through my first alert and thought it might be helpful to others.

### CODE REVIEW

File: docs/source/alert_development_guide.rst [hunk 1]
```diff
@@ -73,6 +73,74 @@ How to get the alert in MozDef?
...
+Next we will need to enable the log and to schedule it. At time of writing this is a bit annoying.
+Open the file
+::
+
+  docker/compose/mozdef_alerts/files/config.py
+
+or simply
+::
+
+  mozdef_alerts/files/config.py
```
Comment (pwnbus) [inline docs/source/alert_development_guide.rst]: This should be `alerts/files/config.py`

Comment (darakian) [reply docs/source/alert_development_guide.rst]: Gotcha, updated.

**Comment by darakian** [comment]
@pwnbus Where would I find the docs on the differences between PhraseMatch/Term[s]Match/etc...?

**Comment by pwnbus** [comment]
@darakian I've got some docs internal that describe what each one is used for, let me try and add them to the repo

**Comment by pwnbus** [comment]
@darakian I created https://github.com/mozilla/MozDef/pull/988 which I think adds what I originally had on hand, with the expectation that these will get improved in the future.

**Comment by darakian** [comment]
@pwnbus thanks!

**Pull request merged**